Wrappers/Python/cil/optimisation/functions/TotalVariation.py

```diff
@@ -120,6 +186,9 @@ def __init__(self,
         # splitting Gradient
         self.split = split
 
+        # Strong convexity for TV
+        self.strong_convexity_constant = strong_convexity_constant
```

Comment:
Is the `strong_convexity_constant` always a positive number? If so, it should be a property that checks what the user input and raises error when wrong.

Reply:
The `strong_convexity_constant` should be a `Number` similar to the `regularisation_parameter`. However, if the`strong_convexity_constant` is 0, then `TotalVariation` and `FGP_TV` remain convex and no acceleration can be used. Also the `regularisation_parameter` can be 0 in theory, however since in the implementation we have this in the denominator an error will be raised. There are many examples in CIL, where we are not doing any type or value checking of these parameters. Maybe we can start doing from this PR or continue rely on the docs.

Reply:
My comments were just on coding and not on maths, clearly. I think a property would be more appropriate as it'll enable you to do the basic check 'is it a non negative number?'

> There are many examples in CIL, where we are not doing any type or value checking of these parameters. Maybe we can start doing from this PR or continue rely on the docs.

That's a fair comment, but I think that we don't have to adhere to bad habits when we see there's room for improvement, nor we have to overhaul the whole of CIL because in many places we haven't used properties.

I'd suggest to make `strong_convexity_constant` a `property`, and continue to use properties or setters (like in PDHG setting[ `sigma` and `tau`](https://github.com/TomographicImaging/CIL/blob/b7e266e3f9d9830ec109488a09ed7c7272361ae0/Wrappers/Python/cil/optimisation/algorithms/PDHG.py#L425-L426)) when setting these type of parameters.

Reply:
Answers from @epapoutsellis are https://github.com/TomographicImaging/CIL/pull/1137/files#r876414786  and https://github.com/TomographicImaging/CIL/pull/1137/files#r876423884

Reply:
We can leave it as it is now until we decide what to do with the whole `Function` class as commented by @epapoutsellis